string-manipulation-2/src/num-vowels.js

```diff
@@ -0,0 +1,12 @@
+/* exported numVowels */
+function numVowels(string) {
+  var smallChar = string.toLowerCase();
+  var count = 0;
+  var splitStr = smallChar.split('');
+  for (var item of splitStr) {
+    if (item === 'a' || item === 'e' || item === 'i' || item === 'o' || item === 'u') {
+      count++;
+    }
+  }
```

Comment:
Stay away from `for...of` loops for now, until you understand how `Symbol` works in JavaScript.